doc/user/OPTIONS.md

```diff
@@ -29,66 +29,66 @@ F3D behavior can be fully controlled from the command line using the following o
 
 ## Window options
 
-| Options                         | Type<br>Default                   | Description                                                                                          |
-| ------------------------------- | --------------------------------- | ---------------------------------------------------------------------------------------------------- |
-| \-\-background-color=\<R,G,B\>  | vector\<double\><br>0.2, 0.2, 0.2 | Set the window _background color_.<br>Ignored if _hdri_ is set.                                      |
-| \-\-resolution=\<width,height\> | vector\<double\><br>1000, 600     | Set the _window resolution_.                                                                         |
-| \-\-position=\<x,y\>            | vector\<double\><br>-             | Set the _window position_ (top left corner) , in pixels, starting from the top left of your screens. |
-| -z, \-\-fps                     | bool<br>false                     | Display a rendering _frame per second counter_.                                                      |
-| -n, \-\-filename                | bool<br>false                     | Display the _name of the file_ on top of the window.                                                 |
-| -m, \-\-metadata                | bool<br>false                     | Display the _metadata_.                                                                              |
-| \-\-hdri-skybox                 | bool<br>false                     | Show the HDRI as a skybox. Overrides \-\-background-color and \-\-no-background.                     |
-| -u, \-\-blur-background         | bool<br>false                     | Blur background.<br>Useful with a HDRI skybox.                                                       |
-| \-\-blur-coc                    | double<br>20                      | Blur circle of confusion radius.                                                                     |
-| \-\-light-intensity             | double<br>1.0                     | _Adjust the intensity_ of every light in the scene.                                                  |
+| Options                         | Type<br>Default               | Description                                                                                          |
+| ------------------------------- | ----------------------------- | ---------------------------------------------------------------------------------------------------- |
+| \-\-background-color=\<R,G,B\>  | color<br>0.2, 0.2, 0.2        | Set the window _background color_.<br>Ignored if _hdri_ is set.                                      |
```

Comment:
It would be great to add a small color swatch when documenting default colors.  

```suggestion
| \-\-background-color=\<R,G,B\>  | color<br><span style="color:rgb(51, 51, 51)">&#9632;</span>0.2, 0.2, 0.2        | Set the window _background color_.<br>Ignored if _hdri_ is set.                                      |
```

Look at https://f3d.app/doc/user/COLOR_MAPS.html#custom-values for example

Reply:
I did but its breaking up a lot the markdown table. Im not sure thats worth it.

Reply:
what do you mean by breaking? The column is just larger?

Reply:
yes, it makes the column super large, not great when viewing the table as raw text